Anime-App/Feature/Anime/View/SettingsViewController.swift

```diff
@@ -0,0 +1,179 @@
+        stackView.axis = .vertical
+        stackView.layer.cornerRadius = 32
+        return stackView
+    }()
+    
+    private lazy var animeImageView: UIImageView = {
+        let animeImageView = UIImageView()
+        animeImageView.image = UIImage(named: "gundam")
+        animeImageView.contentMode = .scaleAspectFit
+        animeImageView.translatesAutoresizingMaskIntoConstraints = false
+        return animeImageView
+    }()
+    
+    private lazy var hapticFeedBackStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        stackView.spacing = 32
+        return stackView
+    }()
+    
+    private lazy var audioStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.alignment = .center
+        stackView.distribution = .fillEqually
+        stackView.axis = .horizontal
+        stackView.spacing = 32
+        return stackView
+    }()
+    
+    private lazy var appVersionLbl: UILabel = {
+        let appVersionLbl = UILabel()
+        appVersionLbl.translatesAutoresizingMaskIntoConstraints = false
+        appVersionLbl.textColor = .black
+        return appVersionLbl
+    }()
+    
+    private lazy var hapticFeedbackLbl: UILabel = {
+        let hapticFeedbackLbl = UILabel()
+        hapticFeedbackLbl.translatesAutoresizingMaskIntoConstraints = false
+        hapticFeedbackLbl.textColor = .black
+        hapticFeedbackLbl.text =  "haptic"
+        hapticFeedbackLbl.font = .init(name: "", size: 14)
+        return hapticFeedbackLbl
+    }()
+    
+    
+    private lazy var audioLbl: UILabel = {
+        let audioLbl = UILabel()
+        audioLbl.translatesAutoresizingMaskIntoConstraints = false
+        audioLbl.textColor = .black
+        audioLbl.text =  "Audio"
+        audioLbl.font = .init(name: "", size: 14)
+        return audioLbl
+    }()
+    
+    private lazy var hapticFeedbackSwitch: UISwitch = {
+        let hapticFeedbackSwitch = UISwitch()
+        hapticFeedbackSwitch.translatesAutoresizingMaskIntoConstraints = false
+        hapticFeedbackSwitch.isOn = false
+        hapticFeedbackSwitch.addTarget(self, action: #selector(toggleHapticFeedback), for: .touchUpInside)
+        return hapticFeedbackSwitch
+    }()
+    
+    private lazy var audioSwitch: UISwitch = {
+        let audioSwitch = UISwitch()
+        audioSwitch.addTarget(self, action: #selector(toggleAudio), for: .touchUpInside)
+        audioSwitch.translatesAutoresizingMaskIntoConstraints = false
+        return audioSwitch
+    }()
+    
+    @objc func toggleHapticFeedback() {
+        if hapticFeedbackSwitch.isOn == true {
+            print("haptic Feedback is on")
+            defaults.set(true, forKey: "UseHaptic")
```

Comment:
When naming keys in user defaults. it's better to go along the lines of `isHapticsEnabled` it's a lot clearer. Can you rename all of your keys to follow this pattern and can you use an enum to organise them all too.